packages/story-editor/src/app/media/utils/useFFmpeg.js

```diff
@@ -323,11 +387,7 @@ function useFFmpeg() {
           type
         );
       } catch (err) {
-        // eslint-disable-next-line no-console -- We want to surface this error.
-        console.log(err);
```

Comment:
Why remove that here?

Reply:
Re-added

https://github.com/GoogleForCreators/web-stories-wp/pull/12335/files/4de45e185d683c1b89da934e9685baf6e095bea8#diff-8383cf154a6a03f61cb68645b7fd4267892b7df793c6b3cceb999e40bcf2522bR333

Reply:
I think I was wrong. Console error are fine in this file.